internal/helpers/schemaregistry/schema_registry.go

```diff
@@ -0,0 +1,82 @@
+package schemaregistry
```

Comment:
since you changed the package, you also have to adapt places where it is used

Reply:
Riiiight. I'll do it